src/movie_buddy/utils.py

```diff
@@ -19,7 +20,7 @@ def add_sentences(data: pd.DataFrame, sentences: list[str], *, encoder) -> pd.Da
     return add_umap_to_dataset(all_sentences, reduced_encodings)
 
 
-def plot_movies(data: pd.DataFrame, *, sample: int = 5000) -> None:
```

Comment:
We realised a sampling parameter would make the viz faster, but if you noticed you don't need it that's ok to remove.

Reply:
Yes, but it was not used in the function :)